Assets/WebGLTemplates/AirConsole-2020/airconsole-unity-plugin.js

```diff
@@ -355,8 +361,6 @@ App.prototype.processUnityData = function (data) {
         this.airconsole.setCustomDeviceState(data.data);
     } else if (data.action == "setCustomDeviceStateProperty") {
         this.airconsole.setCustomDeviceStateProperty(data.key, data.value);
-    } else if (data.action === "enablePlayerSilencing") {
```

Comment:
Is it normal that it is removed?

Reply:
Yes. It never was in use - it was originally added during the initial phase of the 2.5.0 efforts but we ultimately decided on going a different path (AirConsole init time where the entry in the airconsole-settings.js stems from) removing any runtime API.